vendor/github.com/docker/distribution/registry/client/blob_writer.go

```diff
@@ -99,7 +99,7 @@ func (hbu *httpBlobUpload) Write(p []byte) (n int, err error) {
 		return 0, fmt.Errorf("bad range format: %s", rng)
 	}
 
-	return (end - start + 1), nil
```

Comment:
MultiReader can't use Copy to get ReadFrom.  It's likely few people have ever streamed this content.